src/libraries/Common/src/System/Drawing/KnownColor.cs

```diff
@@ -164,6 +164,7 @@ enum KnownColor
         Plum,
         PowderBlue,
         Purple,
+        RebeccaPurple,
```

Comment:
This is also public, you need to add it to the ref as well:

https://github.com/dotnet/runtime/blob/master/src/libraries/System.Drawing.Primitives/ref/System.Drawing.Primitives.cs#L193

Reply:
i have added it 👍